src/common/password.cpp

```diff
@@ -251,7 +244,7 @@ namespace tools
     login out{};
     password_container wipe{std::move(userpass)};
 
-    const auto loc = wipe.password().find(':');
+    const auto loc = userpass.find(':');
```

Comment:
This is now busted, `userpass` has been moved.

Reply:
Oooh, thanks!